src/network/networkd-link.c

```diff
@@ -1643,6 +1654,12 @@ static int link_set_bridge(Link *link) {
                         return log_link_error_errno(link, r, "Could not append IFLA_BRPORT_PRIORITY attribute: %m");
         }
 
+        if (link->network->multicast_router != _MULTICAST_ROUTER_INVALID) {
+                r = sd_netlink_message_append_u16(req, IFLA_BRPORT_MULTICAST_ROUTER, link->network->multicast_router);
```

Comment:
this should be `_append_u8()`. See https://github.com/torvalds/linux/blob/fe5cdef29e41c8bda8cd1a11545e7c6bfe25570e/net/bridge/br_netlink.c#L673